cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/LocalManager.java

```diff
@@ -107,6 +111,7 @@ public LocalManager(CConfiguration configuration, PipelineFactory pipelineFactor
     pipeline.addLast(new DeployDatasetModulesStage(datasetFramework));
     pipeline.addLast(new CreateDatasetInstancesStage(datasetFramework));
     pipeline.addLast(new CreateStreamsStage(id, streamAdmin, exploreFacade, exploreEnabled));
+    pipeline.addLast(new CreateSchedulesStage(store, scheduler));
```

Comment:
We need to still have the app specification of the old app in the store, so that we can compare which schedules are new, and which schedules are to be updated.